src/contrib/persistence/Akka.Persistence.Sqlite/sqlite.conf

```diff
@@ -89,4 +89,27 @@
 			write-isolation-level = unspecified
 		}
 	}
-}
\ No newline at end of file
+}
+
+akka.persistence.query.journal.sql {
```

Comment:
Inlined the `akka.persistence.query.journal.sql` into the `Akka.Persistence.Sqlite/reference.conf` to maintain backwards compat for Sqlite users.